domain_source/domain_source.module

```diff
@@ -138,7 +141,7 @@ function domain_source_form_alter(&$form, &$form_state, $form_id) {
 /**
  * Validate form submissions.
  */
-function domain_source_form_validate($element, \Drupal\Core\Form\FormStateInterface $form_state) {
+function domain_source_form_validate($element, FormStateInterface $form_state) {
```

Comment:
Do we have to type hint $element?

Reply:
FormValidatorInterface::executeValidateHandlers does not typehint the &$form array, but it is assumed to be an array so I added a type cast to make this explicit